src/journeys_utils.js

```diff
@@ -449,22 +451,16 @@ journeys_utils.animateBannerEntrance = function(banner) {
 
 /***
  * @function journeys_utils.findDismissPeriod
- * @param {string} html
+ * @param {Object} metadata
  *
  * Checks template metadata to determine how long to not show banner when dismissed
  */
-journeys_utils.findDismissPeriod = function(html) {
+journeys_utils.findDismissPeriod = function(metadata) {
 	// default dismiss period to 1 week
 	var dismissPeriod = 7;
```

Comment:
don't we want this to be in seconds now?

Reply:
This is a dead method. My bad. I'll remove this.